policyengine_us/variables/gov/local/ca/riv/liheap/ca_riv_liheap_eligible.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ca_riv_liheap_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "Eligible for the California Riverside County LIHEAP"
+    definition_period = YEAR
+    defined_for = "in_riv"
+    reference = "https://capriverside.org/utility-assistance-program"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.hhs.liheap
+        state_median_income = spm_unit("hhs_smi", period)
+        # The income concept is not clearly defined, assuming IRS gross income
+        income = add(spm_unit, period, ["irs_gross_income"])
```

Comment:
Can you use this for the income definition:
https://capriverside.org/sites/g/files/aldnop136/files/2024-12/2025%20LIHEAP%20CAP%20APPLICATION%20ENGLISH.pdf